interface-impl-amf/src/main/java/org/mule/amf/impl/model/ParameterImpl.java

```diff
@@ -231,8 +236,16 @@ private static Boolean needsQuotes(Shape anyShape) {
     return !(NUMBER_DATA_TYPES.contains(dataType) || BOOLEAN_DATA_TYPE.equals(dataType));
   }
 
-  @Override
-  public boolean validateArray(Collection<?> values) {
-    return validate(getArrayAsYamlValue(this, values));
+  private static boolean areArrays(UnionShape unionShape) {
+    boolean areArrays = false;
```

Comment:
I would prefer a more explicit name like `hasAnArrayVariant`.

Or a comment at the start of the function "returns true if the union admits an array variant and all the variants are either null or array types"

Reply:
thanks for the hint on the name, love it.